src/issue.js

```diff
@@ -95,11 +119,16 @@ async function createOrUpdateIssue(
   registerHandlebarHelpers({
     commitMessageLines,
   })
+
+  const newVersion = await getAutoBumpedVersion()
+
   const issueBody = await renderIssueBody({
     commits: unreleasedCommits,
     latestRelease,
     notifyAfter,
+    newVersion,
```

Comment:
I wouldn't call this newVersion, because it's not a version, it's a semver bump type